www/js/lib/uiUtil.js

```diff
@@ -211,7 +211,28 @@ define([], function() {
         else 
             return rect.top < window.innerHeight && rect.bottom > 0 && rect.left < window.innerWidth && rect.right > 0;
     }
+    /**
+     * Encodes the html escape characters in the string before using it as html class name,id etc.
+     * 
+     * @param {String} string The string in which html characters are to be escaped
+     * 
+     */
+    function htmlEscapeChars(string){
+        var escapechars = {
+            '&': '&amp;',
+            '<': '&lt;',
+            '>': '&gt;',
+            '"': '&quot;',
+            "'": '&#39;',
+            '/': '&#x2F;',
+            '`': '&#x60;',
+            '=': '&#x3D;'
+          };
+          string = String(string).replace(/[&<>"'`=\/]/g, function (s) {
```

Comment:
Sorry for another minor remark : CodeFactor says the backslash is unnecessary here : https://www.codefactor.io/repository/github/kiwix/kiwix-js/pull/578

Reply:
Ya sir...I have just removed that.....please review now.